recipe/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -ex
+
+# necessary for cross-compilation to point to the right env
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+
+# HACK: extend $CONDA_PREFIX/meson_cross_file that's created in
+# https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/main/recipe/activate-gcc.sh
+# https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/main/recipe/activate-clang.sh
+# to use host python; requires that [binaries] section is last in meson_cross_file
+echo "python = '${PREFIX}/bin/python'" >> ${CONDA_PREFIX}/meson_cross_file.txt
```

Comment:
Do other projects like scipy need similar hacks? Should this problem be reported/cross linked with issues in the `*-compiler-activation-feedstock` repos?

Reply:
Yes I took it from numpy and scipy feedstock